validator/validator.protoascii

```diff
@@ -2373,6 +2373,30 @@ tags: {  # amp-pinterest
     }
   }
 }
+tags: {  # amp-reach-player
```

Comment:
It's probably going to cause us some merge hassle. Let me grab this change and pull it to github, then we can revert this one file. Is that OK?